lib/ansible/modules/cron.py

```diff
@@ -575,6 +578,9 @@ def main():
         mutually_exclusive=[
             ['insertafter', 'insertbefore'],
         ],
+        required_one_of=[
+            ['name', 'cron_file']
```

Comment:
The `name` parameter above has `required=True` and upon a closer look, being optional went through a deprecation period already and was removed a couple weeks ago in https://github.com/ansible/ansible/commit/d7df8a4484e50b27e921e9fe887dc84c86617e7f.